test/chunking-form/samples/basic-chunking/_expected/system/main1.js

```diff
@@ -0,0 +1,23 @@
+System.register(['./chunk1.js'], function (exports, module) {
+  'use strict';
+  var fn;
+  return {
+    setters: [function (module) {
+      fn = module.fn;
+    }],
+    execute: function () {
+
+      function fn$1 () {
+        console.log('dep1 fn');
+      }
+
+      class Main1 {
+        constructor () {
+          fn$1();
+          fn();
+        }
+      } exports('default', Main1);
```

Comment:
Wouldn't this look better if `exports` was on a new line? It seems to be valid syntax but it sure looks strange.

Reply:
The issue I found with this is that adding a new line results in the indentation method of MagicString not applying properly, and I didn't want to get into fixing that. I also thought this seems like some kind of nice shorthand anyway.